src/com/content-item/item.js

```diff
@@ -206,6 +205,24 @@ export default class ContentItem extends Component {
 		}
 	}
 
+	onAnonymousComments () {
+		const anon = this.state.allowAnonymous;
+		const node = this.props.node;
+		let update = null;
+		if (anon) {
+			update = $NodeMeta.Remove(node.id, {'allow-anonymous-comments': 0});
+		}
+		else {
+			update = $NodeMeta.Add(node.id, {'allow-anonymous-comments': 1});
+		}
+
+		return update.then(r => {
+				if ( r && r.changed ) {
+					this.setState({allowAnonymous: !anon});
```

Comment:
Please wrap member names in string quotes. i.e. `{'allowAnonymous': ...}`. For consistency with JSON.